pages/_app.js

```diff
@@ -0,0 +1,12 @@
+import "../styles/globals.css";
+import localFont from "@next/font/local";
```

Comment:
you can move these configuration out side the app.js. may be you can put it in config dir.